src/schema/TypeORMConverter.ts

```diff
@@ -40,7 +40,16 @@ export function filenameToImportPath(filename: string): string {
 }
 
 export function extractEnumObject(column: ColumnMetadata): GraphQLEnumType {
```

Comment:
Could this whole thing just be:

```js
const modelEnums = column.entityMetadata.inheritanceTree.map(f => 
  getMetadataStorage()
  .getEnum(modelNames[f.name], column.propertyName)
);
return modelEnums.find(e => Boolean(e));
```

admittedly have not run this, just wondering if `for` loop with break can be removed here.

Reply:
my C / C++ was showing, wasn't it?

Reply:
your version was definitely prettier (and didn't include `break` or `goto`)

Reply:
Awesome :-)